src/util.js

```diff
@@ -0,0 +1,68 @@
+// generate manager card
+const generateManagerCard = (manager) => {
+  return `<div class="col">
+    <div class="card" style="width: 18rem">
+        <div class="card-body">
+        <h5 class="card-title">${manager.name}</h5>
+        <h6 class="card-subtitle mb-2 text-muted">Manager 👨‍🏫</h6>
+        <h6 class="card-subtitle mb-2 text-muted">${manager.id}</h6>
+        <p class="card-text">
+            If you'd like to get in touch or have issues regarding
+            escalations, please reach out to me below.
+        </p>
+        <div class="d-flex flex-row flex-wrap justify-content-center">
+            <a href="${manager.email}" class="card-link text-decoration-none">${manager.email}</a>
```

Comment:
Please add `mailto:` links for the email, and a `href` to the actual GitHub profile URL for the given username.